doc/radosgw/config-ref.rst

```diff
@@ -7,6 +7,11 @@ The following settings may added to the Ceph configuration file (i.e., usually
 settings may contain default values. If you do not specify each setting in the
 Ceph configuration file, the default value will be set automatically.
 
+Configuration variables set under the ``[client.radosgw.{instance-name}]``
+section will not apply to rgw or radosgw-admin commands without an instance-name
+specified in the command. Thus variables meant to be applied to all RGW
+instances or all radosgw-admin commands can be put into the ``[global]`` or the
```

Comment:
@alimaredia if the settings specified in `[client.radosgw]` does not apply to rgw or radosgw-admin commands without an instance-name specified in the command, and the ones specified in `[global]` do, we certainly need to emphasize that they should go to `[global]`. if that's the case, i am good with this change. otherwise, i think it would be better to limit the scope of the settings into `[client.radosgw]`.

anyway,  i don't feel strong either way. so lgtm.